tests/jest/ext.wikibase.facetedsearch.test.js

```diff
@@ -0,0 +1,19 @@
+describe( 'ext.wikibase.facetedsearch.js', () => {
+	// TODO: Add more tests when the JS is more stable
```

Comment:
Would it be able to add a stub test for the actual JS file in the meantime? It could just load the JS and without the necessary HTML `init()` should just return. Just as an example, since it's unclear how the current JS will be tested.

Reply:
> Would it be able to add a stub test for the actual JS file in the meantime? It could just load the JS and without the necessary HTML `init()` should just return. Just as an example, since it's unclear how the current JS will be tested.
> 
> Otherwise if you have tests almost ready on an incoming PR, then ignore this.

In the current JS, the functions aren't exported so it can't be accessed by the test. I am going to refactor the JS for #112 after we gets the other JS-related tasks in the MVP milestone.